mosquitto/docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+services:
```

Comment:
This docker-compose file and the umbrel-app manifest file need to have the `.yml` suffix instead of `.yaml` ,otherwise they won't render in the app store and you won't be able to install them via the UI or CLI. Were you still able to install it somehow?